relay-dynamic-config/src/global.rs

```diff
@@ -156,6 +156,20 @@ pub struct Options {
     )]
     pub metric_stats_rollout_rate: f32,
 
+    /// Overall sampling of span extraction.
+    ///
+    /// This number represents the fraction of transactions for which
+    /// spans are extracted. It applies on top of [`crate::Feature::ExtractSpansAndSpanMetricsFromEvent`],
+    /// so both feature flag and sample rate need to be enabled to get any spans extracted.
+    ///
+    /// Any value below 1.0 will cause the product to break, so use with caution.
```

Comment:
```suggestion
    /// Note: Any value below 1.0 will cause the product to break, so use with caution.
```

In my opinion it's a little bit more visible, harder to miss